src/RapidezStatamicServiceProvider.php

```diff
@@ -249,6 +251,17 @@ public function bootSitemaps(): static
         return $this;
     }
 
+    public function bootStaticCaching(): static
+    {
+        if (config('statamic.static_caching.invalidation.class')) {
+            return $this;
+        }
+        
+        config()->set('statamic.static_caching.invalidation.class', CustomInvalidator::class);
+
+        return $this;
+    }
```

Comment:
Why not add this in the config file? then it can be overwritten too, now it's not that clear that it's possible to overwrite this class

Reply:
I don't feel like we should make some config for this, since it's already basic Statamic config.
I'd rather add some more documentation for this in https://docs.rapidez.io/3.x/packages/statamic.html.